ops.go

```diff
@@ -79,47 +79,26 @@ func Tail[T any](num int, arr []T) []T {
 }
 
 // Drop allocates a new slice, with the first `num` elements dropped.
-func Drop[T any](num int, arr []T) []T {
-	if len(arr) < num {
+func Drop[T any](num uint, arr []T) []T {
+	l := uint(len(arr))
+	if l < num {
 		return []T{}
 	}
 
-	newLen := len(arr) - num
-	slice := make([]T, 0, newLen)
-	slice = append(slice, arr[num:]...)
+	slice := make([]T, l-num)
+	copy(slice, arr[num:])
 
 	return slice
 }
 
-// DropString allocates a new string, with the first `num` bytes of a string dropped.
-func DropString(num int, what string) string {
-	if len(what) < num {
-		return ""
-	}
-
-	newLen := len(what) - num
-	buffer := make([]byte, 0, newLen)
-	buffer = append(buffer, what[num:]...)
-
-	return string(buffer)
-}
-
-// Skip skips the first `num` elements.
-func Skip[T any](num int, arr []T) []T {
-	if len(arr) < num {
-		return []T{}
+// Skips skips the first `num` elements by slicing (underlying array unaffected).
+func Skip[T any](num uint, arr []T) []T {
```

Comment:
Huh, I didn't even know you could return zero slice by keeping the reference. Kudos for finding this and saving an allocation!